pkg/kapp/clusterapply/add_or_update_change.go

```diff
@@ -143,7 +145,13 @@ func (c AddOrUpdateChange) tryToResolveUpdateConflict(
 			return fmt.Errorf("Expected recalculated change to be an update")
 		}
 		if recalcChanges[0].OpsDiff().MinimalMD5() != c.change.OpsDiff().MinimalMD5() {
-			return fmt.Errorf(errMsgPrefix+"(approved diff no longer matches): %s", origErr)
+			errMsg := fmt.Sprintf(errMsgPrefix+"(approved diff no longer matches): %s", origErr)
+
+			textDiff, err := recalcChanges[0].ConfigurableTextDiff().Masked(c.diffMaskRules)
+			if err != nil {
+				return fmt.Errorf(errMsg)
+			}
+			return fmt.Errorf(errMsg+": Recalculated diff:\n%s", textDiff.MinimalString())
```

Comment:
nit: Refactor to `fmt.Errorf("%s: Recalculated diff:\n%s", errMsg, textDiff.MinimalString())`

Reply:
Done